### PR TITLE
Fix ui tenant wait after creation

### PR DESCRIPTION
**Pull request by azgabur**
Needs #295
Expected merge conflict in 'testsuite/tests/ui/conftest.py' on line 178 after merging #295 I will solve it after with rebase.

edit: conflict solved

### CODE REVIEW

File: testsuite/tests/ui/conftest.py [hunk 1]
```diff
@@ -188,7 +211,7 @@ def _custom_ui_tenant(username: str = "",
 
         account = resilient.resource_read_by_name(master_threescale.accounts, organisation)
         tenant = master_threescale.tenants.read(account.entity_id)
-        tenant.wait_tenant_ready()
+        _ui_wait_tenant_ready(tenant)
```
Comment (jsmolar) [inline testsuite/tests/ui/conftest.py:214]: What happens when `_ui_wait_tenant_ready(tenant)` returns False?

Comment (azgabur) [reply testsuite/tests/ui/conftest.py]: Will continue anyways. 
I put assert before it, so it wont happen.

Comment (azgabur) [reply testsuite/tests/ui/conftest.py:209]: Changed logic of the function. Now raises `AssertationError` if the backoff fails on last try.

Comment (azgabur) [reply testsuite/tests/ui/conftest.py]: Reverted the True/False logic, now it uses logging tool to log reasons for backoff. Tell me if its good idea.

File: testsuite/tests/ui/conftest.py [hunk 2]
```diff
@@ -152,26 +154,54 @@ def devel_login(account, custom_devel_login):
...
+            accounts = navigator.navigate(AccountsView)
+            if accounts.table.row()[0].text == 'No results.':
+                logger.warning("No account found")
+                return False
+            applications = navigator.navigate(ApplicationsView)
+            if applications.table.row()[0].text == 'No results.':
+                logger.warning("No application found")
+                assert False
+            services = navigator.navigate(ProductsView)
+            if services.table.row()[0].text == 'No results.':
```
Comment (jsmolar) [inline testsuite/tests/ui/conftest.py:180]: Le dot

File: testsuite/tests/ui/conftest.py [hunk 3]
```diff
@@ -152,26 +154,46 @@ def devel_login(account, custom_devel_login):
 
 # pylint: disable=unused-argument, too-many-arguments
 @pytest.fixture(scope="module")
-def custom_ui_tenant(master_login, navigator, threescale, testconfig, request, master_threescale):
+def custom_ui_tenant(master_login, navigator, threescale, testconfig, request, master_threescale, browser):
     """Parametrized custom Tenant created via UI"""
 
+    @backoff.on_predicate(backoff.constant, lambda ready: not ready, interval=6, max_tries=5, jitter=None,
+                          on_backoff=lambda _: browser.selenium.refresh())
+    def _backoff(func):
```
Comment (jsmolar) [inline testsuite/tests/ui/conftest.py:162]: change the name to something more meaningful and add documentation

Comment (azgabur) [reply testsuite/tests/ui/conftest.py]: Is it better?

**Review by dhlavac — APPROVED** [review]

**Review by jsmolar — APPROVED** [review]

**Pull request merged**